QUARKUS-508.md

```diff
@@ -0,0 +1,35 @@
+# QUARKUS-508 Create one or more quarkus devfile v2, ensure odo based workflow
+
+Jira: https://issues.redhat.com/browse/QUARKUS-508
+
+The goal of QUARKUS-508 is to provide a runtime descriptor for OpenShift development tools (e.g. odo, Eclipse Che).
+For purposes of this feature request, odo based workflow will be considered.
+
+## Scope of the testing
+
+- Define the scope of the testing
+- Analyze the possible impact on Quarkus itself - e.g performance, compatibility
+- Determine impact on test suites - e.g. new test suite, increased complexity of existing test suite, maintenance burden
+- Determine impact on QE infrastructure - e.g. increased complexity of existing automation, maintenance burden, need for new external resources
+
+### Impact on testsuites and testing automation:
+A deployment strategy utilising `odo` tool will be implemented in [`quarkus-test-framework`](https://github.com/quarkus-qe/quarkus-test-framework/).
+A test using the `odo` deployment strategy will deploy a simple REST endpoint and verify that the endpoint responds.
+This test will be implemented in [`quarkus-test-suite`](https://github.com/quarkus-qe/quarkus-test-suite/) project.
+
```

Comment:
Quarkus tests should be executed when `odo` tool performs new release too.
We should start discussion with the counter parts, please ping me to give you some names

Reply:
And also mention this in the test plan

Reply:
I am not sure I agree on this part, we don't test new releases of `oc` for example.

`odo`, on the RH side, is provided as part of OpenShift dev tools, tied to OCP release.

Reply:
We will test when we have the release. The point if this note is that the odo team should test scenario with Quarkus when they do the release of odo to early notice possible issues.
We should at least start discussion as part of this RFE.